mlflow/pipelines/regression/v1/pipeline.py

```diff
@@ -523,6 +578,19 @@ def read_dataframe(artifact_name, output_dir, file_name, step_name):
                 log_artifact_not_found_warning("registered_model_version", register_step.name)
                 return None
 
+        elif artifact_name == "batch_scoring_data":
+            return read_dataframe(
+                "batch_scoring_data",
+                ingest_scoring_output_dir,
+                IngestScoringStep._DATASET_OUTPUT_NAME,
+                ingest_scoring_step.name,
+            )
```

Comment:
`"batch_scoring_data"` reads to me like the output the entire Batch Scoring DAG. Perhaps we can rename this? `"ingested_scoring_data"` is an idea

Reply:
Agree.  Let me fix this.